src/controllers/feedbackController.js

```diff
@@ -0,0 +1,37 @@
+import Response from '../utils/response';
+import FeedbackServices from '../repositories/feedbackRepository';
+import UserService from '../repositories/userRepository';
+import AccommodationService from '../repositories/accommodationRepository';
+
+class AccommodationFeedbackController {
+  /**
+   * Creates a new feedback.
+   * @param {object} req request
+   * @param {object} res response
+   * @param {object} next next
+   * @returns {object} response object
+   */
+  static async addFeedback(req, res, next) {
+    try {
+      const { feedback } = req.body;
+      const accommodationID = Number(req.params.id);
+      const AccommodationExists = await AccommodationService.findAccommodationById(accommodationID);
+      if (!AccommodationExists) {
+        const response = new Response(res, 404, 'Accommodation not found for your feedback');
+        return response.sendErrorMessage();
+      }
+      const user = await UserService.findByUserId(req.userData.id);
+      const newFeedback = await FeedbackServices.addFeedback({
```

Comment:
@KarangwaEmmy, Good. However I have been providing this feedback since the very beginning. why not handling exceptions/errors here. what do you believe will happen if the addFeeback fails?